pybaseball/analysis/trajectories/utils.py

```diff
@@ -0,0 +1,57 @@
+import numpy as np
+from functools import lru_cache
+from pybaseball.analysis.trajectories.unit_conversions import (
+    REVS_PER_MINUTE_TO_RADIANS_PER_SECOND,
+)
+
+
+@lru_cache(maxsize=64)
+def spin_components(
+        spin: float, spin_angle: float, launch_angle: float, launch_direction_angle: float) -> np.array:
+    """
+    given spin in revolutions per minute, spin angle in degrees,
+    launch angles in degrees, 
+    returns the x,y, z components of spin
+
+    :param spin:
+    :param spin_angle:
+    :return:
+    """
+
+    spin_angle_in_radians = np.deg2rad(spin_angle)
+    launch_angle_in_radians = np.deg2rad(launch_angle)
+    launch_direction_angle_in_radians = np.deg2rad(launch_direction_angle)
```

Comment:
can we remove the `_in_radians`?
Just `spin_angle = np.deg2rad(spin_angle)` etc?
It'll make it cleaner I think

Reply:
hmm, I don't like this for two reasons

1. I prefer not to overwrite existing variables
2. I prefer explicit variable names

I'd argue since this is a util function that's not really public facing, and it's functional, let's just leave it as is.

Reply:
Fair enough